doc/USER_GUIDE.md

```diff
@@ -41,6 +41,7 @@ Users have the following options for specifying their own values:
 | `creator` | ```{{env `USER`}}``` |  |
 | `docker_version` | ```20.10.23-1.amzn2.0.1``` |  |
 | `encrypted` | ```false``` |  |
+| `enable_fips` | ```false``` |  |
```

Comment:
Can you add a description for this template variable?

I also think we should remove the bit from the main README, we don't highlight any other variables there (like `cache_container_images`).